_component_design/tables-basic.md

```diff
@@ -10,9 +10,158 @@ resource: true
+![]({{site.url}}/assets/images/components/tables/basic-tables/tables-hero.svg){:width="100%"}
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+## Default layout
+{: .hxSectionTitle}
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+- Include the check-box icon if a user needs to select or manipulate data within a table. This will always be the first element in a table row, see [table controls]({{site.baseurl}}/components/tables-controls.html)
+- Include the action-cog icon if a user requires the trigger two or more actions on a row.
+- Text justification within each cell is content specific.
+  - Text should be left justified.
+  - Icons, “&mdash;” (null/undefined), and “n/a” (not applicable) should be center justified.
+  - Numbers should be right justified.
+- Only use units in row if they differ row by row and cannot be standardized. Units in body rows appear without parentheses.
+- Truncated text should appear in a [tooltip]({{site.baseurl}}/components/tooltip.html), enabling display on hover or click.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Default table specifications"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/tables/basic-tables/tables-default.svg){:width="100%"}
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+## Condensed table
+{: .hxSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+The use of a condensed table provides for views that display either many or few results. 
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Condensed table specifications"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/tables/basic-tables/tables-condensed.svg){:width="100%"}
+{% endfigure %}
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+### Two-row / small tables
+{: .hxSubSectionTitle}
+
+For instance, when a small amount of information, that is best presented in a tabular format, is presented, the condensed table format allows for a concise presentation. 
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Example: IP addresses currently applied in a cloud server."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/tables/basic-tables/tables-small-condensed.svg){:width="100%"}
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+## Table states 
+{: .hxSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+### Loading
+{: .hxSubSectionTitle}
+
+As tables load, use a gray placeholder to represent the loading text. This pattern informs the user that the data is present but not yet available.
```

Comment:
check this against what is currently defined in the skeleton loading doc.  I think the bars have rounded corners there.  We may also want to point a tentative link at that article here.

Reply:
Updated the image using elements present in the skeleton loading doc for body content.